packages/sourcecred/src/api/lib/node.js

```diff
@@ -14,6 +14,12 @@ const api: any = {...cloneDeep(base)};
 import {LocalInstance} from "../../api/instance/localInstance";
 api.instance.LocalInstance = LocalInstance;
 
+import {getOriginWriteInstance} from "../../api/instance/writeInstance";
```

Comment:
I've added factory methods here because when I tried to register writeInstance module in `packages/sourcecred/src/api/lib/base.js`, the build was failing with
```
sourcecred: [webpack-cli] ModuleNotFoundError: Module not found: Error: Can't resolve 'fs' in '/storage/projects/sourcecred/sourcecred-sources/node_modules/better-sqlite3/lib'
sourcecred:     at /storage/projects/sourcecred/sourcecred-sources/node_modules/webpack/lib/Compilation.js:925:10
sourcecred:     at /storage/projects/sourcecred/sourcecred-sources/node_modules/webpack/lib/NormalModuleFactory.js:401:22
sourcecred:     at /storage/projects/sourcecred/sourcecred-sources/node_modules/webpack/lib/NormalModuleFactory.js:130:21
sourcecred:     at /storage/projects/sourcecred/sourcecred-sources/node_modules/webpack/lib/NormalModuleFactory.js:224:22
sourcecred:     at /storage/projects/sourcecred/sourcecred-sources/node_modules/neo-async/async.js:2830:7
sourcecred:     at /storage/projects/sourcecred/sourcecred-sources/node_modules/neo-async/async.js:6877:13
sourcecred:     at /storage/projects/sourcecred/sourcecred-sources/node_modules/webpack/lib/NormalModuleFactory.js:214:25
sourcecred:     at /storage/projects/sourcecred/sourcecred-sources/node_modules/enhanced-resolve/lib/Resolver.js:213:14
sourcecred:     at /storage/projects/sourcecred/sourcecred-sources/node_modules/enhanced-resolve/lib/Resolver.js:285:5
sourcecred:     at eval (eval at create (/storage/projects/sourcecred/sourcecred-sources/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
sourcecred:     at /storage/projects/sourcecred/sourcecred-sources/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:44:7
sourcecred:     at /storage/projects/sourcecred/sourcecred-sources/node_modules/enhanced-resolve/lib/Resolver.js:285:5
sourcecred:     at eval (eval at create (/storage/projects/sourcecred/sourcecred-sources/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
sourcecred:     at /storage/projects/sourcecred/sourcecred-sources/node_modules/enhanced-resolve/lib/Resolver.js:285:5
sourcecred:     at eval (eval at create (/storage/projects/sourcecred/sourcecred-sources/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:27:1)
sourcecred:     at /storage/projects/sourcecred/sourcecred-sources/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:67:43
sourcecred: resolve 'fs' in '/storage/projects/sourcecred/sourcecred-sources/node_modules/better-sqlite3/lib'
sourcecred:   Parsed request is a module
sourcecred:   using description file: /storage/projects/sourcecred/sourcecred-sources/node_modules/better-sqlite3/package.json (relative path: ./lib)
sourcecred:     Field 'browser' doesn't contain a valid alias configuration
sourcecred:     resolve as module
sourcecred:       /storage/projects/sourcecred/sourcecred-sources/node_modules/better-sqlite3/lib/node_modules doesn't exist or is not a directory
sourcecred:       /storage/projects/sourcecred/sourcecred-sources/node_modules/node_modules doesn't exist or is not a directory
sourcecred:       /storage/projects/sourcecred/node_modules doesn't exist or is not a directory
sourcecred:       /storage/projects/node_modules doesn't exist or is not a directory
sourcecred:       /storage/node_modules doesn't exist or is not a directory
sourcecred:       /node_modules doesn't exist or is not a directory
...
```
presumably because of specificity in the `webpack.config.api.js` configuration for `base.js`. Unsure if that is right or wrong, just noting.